src/Uno.UI/UI/Xaml/DependencyObjectStore.Binder.cs

```diff
@@ -229,15 +224,18 @@ public void ResumeBindings()
 
 		private void BinderDispose()
 		{
-			if (_isDisposed)
+			lock (_gate)
```

Comment:
Could this be done with `Interlocked` instead?

Reply:
+1

Reply:
I'll keep it as-is for now, if this becomes a performance issue, we'll adjust it.